pages/account/responsibility-sharing/dedicated-servers/guide.en-gb.md

```diff
@@ -0,0 +1,168 @@
+| Buy and hold licences and usage rights for the OS used | RA |  |
+| Deploy initial network configuration to devices | I | RA |
+
+#### 2.2. Reversibility model
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+
+#### 2.3. Customers information system setup
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Install the OS | RA | RI |
+| Install the software on your dedicated server | RA |  |
+
+### 3. Service usage
+
+#### 3.1. Operations
+
+##### **3.1.1. Daily operations**
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Ensure network accessibility of the Dedicated Server |  | RA |
+| Manage the software installed on the Dedicated Server | RA |  |
+| Decide to add/remove an option on the Dedicated Server, request the replacement of defective hardware on the Dedicated Bare Server | RA | I |
+| Add/remove options/hardware on the Dedicated Server | I | RA |
+| Adapt the OS configuration to the hardware resources in use | RA |  |
+
+##### **3.1.2. Access management**
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Manage access and security policy for Service users | RA |  |
+| Manage physical and logical access to infrastructures for OVHcloud teams | I | RA |
+| Manage access to the manager | RA | I |
+
+##### **3.1.3. Monitoring**
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Monitor the proper functioning of physical devices (utilities) that are part of the Dedicated Server | I | RA |
+| Monitor physical resource performance | RA | R |
+| Process and clear alarms from Dedicated Server managed devices | I | RA |
+| Keep logs of the information system hosted on the Dedicated Server | RA |  |
+| Keep logs of the Control Plane that manages the Dedicated Server |  | RA |
+| Monitor the performance of services installed on the Dedicated Server | RA |  |
+
+##### **3.1.4. Storage**
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Create, modify, control, restore, delete backup jobs | RA |  |
+| Manage content hosted on infrastructures | RA |  |
+| Ensure data continuity and sustainability | RA |  |
+| Carry out maintenance on the storage and backup devices provided by OVHcloud |  | RA |
+
+##### **3.1.5. Connectivity**
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Ensure that automatic network management systems works (architecture, implementation, software and hardware maintenance for deployed public and private networks, primary IP of dedicated server) | I | RA |
+| Manage IP addressing plan | RA | I |
+
+##### **3.1.6. Management**
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Ensure the security of management infrastructures (API, control pane) | R | RA |
```

Comment:
Is it "control pane" ? "control panel" ? "control plane" ?

Reply:
control plane